docker-compose.yml

```diff
@@ -71,38 +59,41 @@ services:
       - snjs
 
   syncing-server-js:
-    depends_on:
-      - syncing-server-proxy
-      - syncing-server-ruby
-    image: standardnotes/syncing-server-js
-    entrypoint: ["/tools/wait-for.sh", "syncing-server-ruby", "3000", "./docker/entrypoint.sh", "start-web"]
+    image: standardnotes/syncing-server-js:1.15.11
+    entrypoint: ["./wait-for.sh", "syncing-server-ruby", "3000", "./docker/entrypoint.sh", "start-web"]
```

Comment:
```
"./wait-for.sh", "syncing-server-ruby"
```
takes care of the `depends_on` here, right?

Reply:
`depends_on` just waits for the container to start .. doesn't wait for the actual application in the container to be started